cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -445,9 +449,15 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             event.preventDefault();
             this.selectSpecialExam(true);
         },
+        setOnboardingExam: function(event) {
```

Comment:
this method is identical to `setPracticeExam`; could we rename that function and use it for both cases?